packages/elements/src/internals/machines/sign-in/sign-in.machine.ts

```diff
@@ -247,7 +247,7 @@ export const SignInMachine = setup({
     },
     Start: {
       id: 'Start',
-      tags: 'state:start',
+      tags: ['state:start'],
```

Comment:
XState docs use arrays (https://stately.ai/docs/tags) and might be a good idea to just follow that, too